From c04d5c13deba5e5317eb7df5c978a8a7bb843406 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 1 Mar 2018 18:32:38 +0100
Subject: [PATCH 1/1] efi_loader: bootefi should not depend on CMD_NET

If a network can be used in the EFI environment should not depend on
the availibility of network commands.

Fixes: 3b3ea2c56ec4 (Kconfig: cmd: Make networking command dependent on NET)
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 cmd/bootefi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmd/bootefi.c b/cmd/bootefi.c
index c17fa2ca23..096130e44c 100644
--- a/cmd/bootefi.c
+++ b/cmd/bootefi.c
@@ -42,7 +42,7 @@ static void efi_init_obj_list(void)
 #if defined(CONFIG_LCD) || defined(CONFIG_DM_VIDEO)
 	efi_gop_register();
 #endif
-#ifdef CONFIG_CMD_NET
+#ifdef CONFIG_NET
 	efi_net_register();
 #endif
 #ifdef CONFIG_GENERATE_SMBIOS_TABLE
-- 
2.14.2

